src/devices/Ili9341/Ili9341Imaging.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Drawing;
```

Comment:
Is this a bit premature though? Just considering that issue seems to be mostly focused on device APIs rather than samples, ImageSharp is still in beta, and there's only one example of its use in this repo, vs several for System.Drawing

Reply:
AFAIK ImageSharp has already released 1.0 and is currently the recommended way to use. System.Drawing is officially deprecated - see the remarks here: https://docs.microsoft.com/en-us/dotnet/api/system.drawing?view=dotnet-plat-ext-5.0#remarks

Given that you've been extending my work which at the time we didn't have such guidelines I'm ok with merging System.Drawing version but if you got time porting to ImageSharp would be very appreciated. @Ellerbach any objections?

Reply:
As I wrote, "would be perfect to have this sample using ImageSharp" :-) It means it's not mandatory.

Reply:
Oh man. You had to pull out the "perfect" card. Well now I want to go and make it perfect first :-)

Reply:
> Oh man. You had to pull out the "perfect" card. Well now I want to go and make it perfect first :-)

:-D I'm sorry :-D And thanks again for all the hard work on this PR!